Sources/LawParser/Services/LawParser/LawParser.swift

```diff
@@ -345,8 +350,16 @@ final class LawParser: NSObject, XMLParserDelegate {
         return string.replacingOccurrences(of: "<!\\[CDATA\\[|\\]\\]>", with: "", options: [.regularExpression])
     }
 
-    private func convertToSuperscript(from string: String) -> String {
-        return string
+    private func getDisplayable(_ number: Int, _ index: Int?) -> String {
```

Comment:
`getDisplayable` what? Perhaps a better signature for this method would be `composeDisplayableNumber(from number: Int, and index: Int?) -> String` that way we are not signaling that we are getting the number from somewhere instead we are forming it. And when we are invoking it it would look nicer as well, `composeDisplayableNumber(from: someNumber,  and: some index)`